rcl/test/rcl/allocator_testing_utils.h

```diff
@@ -87,6 +87,80 @@ set_failing_allocator_is_failing(rcutils_allocator_t & failing_allocator, bool s
   ((__failing_allocator_state *)failing_allocator.state)->is_failing = state;
 }
 
+typedef struct time_bomb_allocator_state
+{
+  int count_until_failure;
+} time_bomb_allocator_state;
+
+void * time_bomb_malloc(size_t size, void * state)
```

Comment:
All the functions defined in a header should be marked `static` or `inline`.
In this case it doesn't matter much, as this only used in tests. `static` is fine

Reply:
See 1891ff0